components/supervisor-api/control.proto

```diff
@@ -19,7 +19,5 @@ service ControlService {
 message ExposePortRequest {
   // local port
   uint32 port = 1;
-  // external port if missing the the same as port
-  uint32 target_port = 2;
```

Comment:
a breaking change: https://github.com/gitpod-io/openvscode-server/blob/593afd765ae6824b7faeb6bab3a520595494461e/extensions/gitpod-web/src/extension.ts#L542 ?

Reply:
https://github.com/gitpod-io/openvscode-server/pull/202

Reply:
```suggestion
// external port if missing the the same as port
reserved 2;
```

Reply:
> reserved 2;

done

Reply:
thanks, I will approve https://github.com/gitpod-io/openvscode-server/pull/202 later after we run this PR in prod.